bugsnag/wsgi/middleware.py

```diff
@@ -37,6 +42,8 @@ def __init__(self, application, environ, start_response):
         self.environ = environ
 
         bugsnag.configure_request(wsgi_environ=self.environ)
+        if bottle_present:
+            bugsnag.configure().runtime_versions['bottle'] = bottle.__version__
```

Comment:
Should this be moved to `BugsnagMiddleware.__init__` instead?

It seems like setting the bottle version only needs to happen once, but I think `WrappedWSGIApp.__init__` is called for every request.

Reply:
Good spot - thanks, I hadn't noticed that.